docs/filesystem_layer/stratum1.md

```diff
@@ -1,44 +1,69 @@
 # Setting up a Stratum 1
 
-Setting up a Stratum 1 involves the following steps:
-
-- set up the Stratum 1, preferably by running the Ansible playbook that we provide;
-- request a Stratum 0 firewall exception for your Stratum 1 server;
-- request a `<your site>.stratum1.cvmfs.eessi-infra.org` DNS entry;
-- open a pull request to include the URL to your Stratum 1 in the EESSI configuration.
-
-The last two steps can be skipped if you want to host a "private" Stratum 1 for your site.
-
+The EESSI project provides a number of geographically distributed public Stratum 1 servers that you can use to make EESSI available on your machine(s).
+If you want to be better protected against network outages and increase the bandwidth between your cluster nodes and the Stratum 1 servers,
+you could consider setting up a local (private) Stratum 1 server that replicates the EESSI CVMFS repository.
+This guarantees that you always have a full and up-to-date copy of the entire stack available in your local network. 
 
 ## Requirements for a Stratum 1
 
 The main requirements for a Stratum 1 server are a good network connection to the clients it is going to serve,
-and sufficient disk space. For the EESSI repository, a few hundred gigabytes should suffice, but for production
-environments at least 1 TB would be recommended.
+and sufficient disk space. As the EESSI repository is constantly growing, make sure that the disk space can easily be extended if necessary. 
+Currently, we recommend to have at least 1 TB available.
 
 In terms of cores and memory, a machine with just a few (~4) cores and 4-8 GB of memory should suffice.
 
-Various Linux distributions are supported, but we recommend one based on RHEL 7 or 8.
+Various Linux distributions are supported, but we recommend one based on RHEL 8 or 9.
+
+Finally, make sure that ports 80 and 8000 are open to clients.
+
+
+## Configure the Stratum 1
+
+Stratum 1 servers usually replicate from the Stratum 0 server. 
```

Comment:
"Usually"? If it did not replicate that would be a problem.

Reply:
I hope you were not assuming I meant the first... :stuck_out_tongue: Anyway, reworded it a bit in https://github.com/EESSI/docs/pull/157/commits/2b32fa913c96592307fe89062740dc6f6f793ed0.